openapi.yaml

```diff
@@ -29,15 +29,10 @@ security:
   - OAuth2:
       - read
 
-x-topics:
-  - title: Getting started
-    content:
-      $ref: ./docs/getting-started.md
-
```

Comment:
what is the rational behind this change? Why would we want to remove the getting started topic from the doc?

Reply:
@paulRbr thats just to keep Speakeasy happy as it doesnt understand the extension. Closed this PR as it only existed to make images for an article. 🙌🏻